src/components/MovieForm/MovieFormFields.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+
+const MovieFormFields = ({movie}) => {
+    const {title, release_date, poster_path, genres, overview, runtime} = movie;
+    const formFields = {title, release_date, poster_path, genres, overview, runtime};
+    const formLabels = [
+        {
+            label: 'Title',
+            placeholder: 'Add Movie\'s Name Here',
+        }, {
+            label: 'Release date',
+            placeholder: 'Select Date',
+        }, {
+            label: 'Movie URL',
+            placeholder: 'Movie URL here',
+        }, {
+            label: 'Genre',
+            placeholder: 'Select Genre'
+        }, {
+            label: 'Overview',
+            placeholder: 'Overview here'
+        }, {
+            label: 'Runtime',
+            placeholder: 'Runtime here'
+        }];
+
```

Comment:
you have jsx for this, there is no need to make it more declarative than it's already is